tests/subsys/mpsl/pm/pm_test.c

```diff
@@ -0,0 +1,314 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+#include <unity.h>
+#include <stdbool.h>
+#include <stdio.h>
+
+
+#include <errno.h>
+
+#include "cmock_policy.h"
+#include "cmock_mpsl_pm.h"
+#include "cmock_mpsl_pm_config.h"
+#include "cmock_mpsl_work.h"
+
+#include <mpsl/mpsl_pm_utils.h>
+#include <zephyr/kernel.h>
+
+#define NUMELTS(X) (sizeof((X)) / sizeof(X[0]))
```

Comment:
would this work? https://docs.zephyrproject.org/latest/kernel/util/index.html#c.ARRAY_SIZE

Reply:
Will try :)